js/index.js

```diff
@@ -1 +1,87 @@
 // Your code goes here
+
+
+//mouseover-done
+let logoText = document.querySelector('h1');
+logoText.addEventListener('mouseover', function(event) {
+    this.style.fontSize = '200';
+    this.style.fontFamily = 'Roboto';
+    this.style.color = 'pink';
+    this.innerText = 'Mindy was Here!'
+});
+
+//keydown-done
+let bodyKey = document.querySelector('body')
+bodyKey.addEventListener('keydown', function(event) {
+    this.classList.toggle('changeColor')
+});
+
+//wheel
+let imageIntro = document.querySelector(".intro img");
+imageIntro.addEventListener('wheel' , (event) => {
+    imageIntro.style.display = "none";
+});
+
+//drag/drop-done
+const imgAdventure = document.querySelector(".content-section img")
+imgAdventure.addEventListener('dragstart', (event) => {
+    imgAdventure.style.opacity = 0.3;
+}, false);
+
+//load
+window.addEventListener('load', () => {
+    let textChange = document.querySelector('.home .intro p')
+    textChange.style.background = 'lightgray';
+});
+
+//focus/blur-done
+let welcomP = document.querySelector('.home .intro p')
```

Comment:
This was a cool effect.  Maybe you could have the blur fade out with greensock as the mouse leaves.